Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -2003,13 +2003,20 @@ void NetworkProcessProxy::setEmulatedConditions(PAL::SessionID sessionID, std::o
 
 void NetworkProcessProxy::fetchLocalStorage(PAL::SessionID sessionID, CompletionHandler<void(HashMap<WebCore::ClientOrigin, HashMap<String, String>>&&)>&& completionHandler)
 {
-    if (!canSendMessage()) {
+    if (!canSendMessage())
         completionHandler({ });
```

Comment:
This needs a return statement after.

Reply:
Ah yes, I did misunderstand. That makes sense. I've removed the check.